.gitignore

```diff
@@ -35,4 +35,7 @@ lerna-debug.log*
 !.vscode/extensions.json
 
 # env
-.env
\ No newline at end of file
+.env
+
+# internal scripts
+_*.*
```

Comment:
I need this to be ignored as I have multiple shell script files for automation to speed up some operations, and I don't want to have to stash them whenever I need to shift branches.